src/main/java/mods/eln/transparentnode/turbine/TurbineSlowProcess.kt

```diff
@@ -0,0 +1,24 @@
+package mods.eln.transparentnode.turbine
```

Comment:
This class is not needed anymore by using the new audio looping API, I have it removed in my PR (#602) already, so it is pointless to convert it to Kotlin.